python_src/src/lamp_py/performance_manager/l1_rt_metrics.py

```diff
@@ -639,34 +698,31 @@ def process_metrics_table(
             sa.and_(
                 travel_times_cte.c.trip_stop_hash
                 == dwell_times_cte.c.trip_stop_hash,
-                sa.func.coalesce(dwell_times_cte.c.dwell_time_seconds, 0) > 0,
             ),
             full=True,
         )
         .join(
-            headways_cte,
+            headways_branch_cte,
             sa.and_(
                 sa.func.coalesce(
                     travel_times_cte.c.trip_stop_hash,
                     dwell_times_cte.c.trip_stop_hash,
                 )
-                == headways_cte.c.trip_stop_hash,
-                sa.or_(
-                    sa.func.coalesce(headways_cte.c.headway_branch_seconds, 0)
-                    > 0,
-                    sa.func.coalesce(headways_cte.c.headway_trunk_seconds, 0)
-                    > 0,
-                ),
+                == headways_branch_cte.c.trip_stop_hash,
```

Comment:
why is this in a `sa.and_` block. it looks like a single boolean to me?

Reply:
Yes, these and_ statements are unnecessary, I'll get rid of them.